File: Inventory Management System/Models/Dto/SalesDto.cs
namespace Models.Dto
{
    public class SalesDto : BaseModel<long>
    {
        public string CustomerName { get; set; }
        public string CustomerEmail { get; set; }

        public long ProductId { get; set; }

        public ProductDto Products { get; set; }

        public int Quantity { get; set; }

        public double TotalPrice { get; set; }
    }
}

File: IMS-FE/src/app/core/openapi/models/sales-dto.ts
/* tslint:disable */
/* eslint-disable */
import { ProductDto } from '../models/product-dto';
export interface SalesDto {
  createdDate?: string;
  customerEmail?: string | null;
  customerName?: string | null;
  id?: number;
  productId?: number;
  products?: ProductDto;
  quantity?: number;
  totalPrice?: number;
  updateDate?: string | null;
}
